src/__tests__/AddIngridient.test.js

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import AddIngridient from "../components/AddIngridient";
+import userEvent from "@testing-library/user-event";
+
+describe("AddIngridients", () => {
+  test("Input for ingridient works", async () => {
```

Comment:
I don't think this test make sense. You are testing 3rd party library input standard functionality.